lib/SimpleSAML/Logger/FileLoggingHandler.php

```diff
@@ -121,8 +133,20 @@ public function log(int $level, string $string): void
                 array_push($replacements, date($format));
             }
 
-            $string = str_replace($formats, $replacements, $string);
-            file_put_contents($this->logFile, $string . \PHP_EOL, FILE_APPEND);
+            if (preg_match('/^php:\/\//', $this->logFile)) {
+                // Dirty hack to get unit tests for Windows working.. Symfony doesn't deal well with them.
```

Comment:
See: https://github.com/symfony/symfony/pull/29661#issuecomment-1055651509

Reply:
Because I didn't get a response, I've opened an issue for this: https://github.com/symfony/symfony/issues/45749